### PR TITLE
Set up DB and create users table

### PR DESCRIPTION
**Pull request by annawinther**
# Description

Setting up and connecting the Postgres database and create the users table.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: data/dbConfig.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const knex = require('knex');
+
+// const knexfile = require('../knexfile');
```
Comment (ethaeral) [inline data/dbConfig.js]: Line 3 and 4 can be removed

**Review by emkayDauda — DISMISSED** [review]
Great job @annawinther

**Comment by annawinther** [comment]
Sorry, didn't mean to commit that twice 😄

**Review by ethaeral — DISMISSED** [review]
you're so good anna

**Review by emkayDauda — DISMISSED** [review]

**Review by emkayDauda — APPROVED** [review]

**Review by nobioma1 — APPROVED** [review]
Great Job @annawinther

**Pull request merged**